### DISCUSSION TITLE
preproduced rfid or qr tags

### DISCUSSION THREAD

**Discussion by urapadmin** [arch-kiosk/arch-kiosk-office#1229] [users: ideas for new features]
Moved here from "ideas and drafts for the recording system" in the wiki:

A roll with preprinted unique qr tags could be taken into the field. Whenever a bag is produced it gets the qr tag. At that point these can be normal qr tags (they don't need to last particularly long) since most of the bags will get dumped later anyhow. Those bags which go to storage could get an rfid tag instead. The idea behind it is that the collected material does not get a manual number written on it but the ipad scans the qr tag that is on the bag and connects the record to the bag that way.
Registration then registers the incoming bags with a scanner (can eben be an iphone) and the processural codes are entered in connection with the scanning. Laurel thought that all takes too long, but I think the loss will be compensated by more precision. Of course, everybody (ceramicist, drawing …) has to use scanning devices. Only the devices translate the coded unique number into a meaningfully number we are used to.

**Comment by lbestock** [comment]
My initial concern was with registration and if that really works, having a scanning device in one hand instead of bags in both hands as one moves bags of ceramics around. But now an additional danger I see here is that someone in the field would put the QR code on a bag and forget to scan it. As has been brought up in other recent discussions/tickets, sometimes one has a LOT of CM from the same locus/lot, for instance, all getting individual tags. If I tag 20 bags but only scan 19 of them, the 20th is mush. A bag that is tagged with an as-yet-unassigned tag LOOKS done to an excavator - it is visually indistinguishable from the bags that have been tagged and scanned. But if it hasn't been scanned then it's trash, it comes back from the field and there is no way to reconstruct which context it's from. So thinking through this, I would still want the excavator to write the URAP number on the tag as soon as it has been scanned and given one - that would be a visible indication that the bag was ready to go back to the house. But then it would also be visible to the registrar, who could still sort things without scanning then. Which sort of defeats the purpose? I am willing to try this (next time I have anything to excavate) but it makes me so nervous that I'm clearly devising safety valves that make it hard to really test.

**Reply by urapadmin** [reply]
we also don't have to follow up all ideas. Given that time is what we have least of we should decide against most ideas. I propose this as a candidate.

**Reply by lbestock** [reply]
Why can't I mark a reply as the answer? Because I second your proposal and would mark it as answered (for now).

**Comment by urapadmin** [comment]
only the original author can do that because Github treats this like a support question type of thing. It is very barebones and I am only moderately happy with it. But you can apply the label discussion: closed.
